test/executables/test_components/package_without_match/react_render_with_comment.dart

```diff
@@ -0,0 +1,3 @@
+void main() {
+  // [ ] Check this box upon manual validation that the component rendered by this expression uses a ref safely. This will be removed once the transition to React 16 is complete.
```

Comment:
This is used because if the package version checking didn't bail it would return an exit code of 1.